SpatialGDK/SpatialGDK.uplugin

```diff
@@ -11,15 +11,15 @@
   "MarketplaceURL": "",
```

Comment:
Can't comment properly, but this needs an update to 0.9 in the lines above.
If this is missing from the release process, it needs to be added there.

Reply:
Yeah VersionName and Version both need updating